apps/judicial-system/web/src/routes/CourtOfAppeal/Cases/Cases.tsx

```diff
@@ -159,9 +163,17 @@ const CourtOfAppealCases = () => {
             : { color: 'darkerBlue', text: formatMessage(tables.completedTag) }
 
         return (
-          <Tag variant={tagVariant.color} outlined disabled>
-            {tagVariant.text}
-          </Tag>
+          <>
+            <Box marginRight={1} marginBottom={1}>
+              <Tag variant={tagVariant.color} outlined disabled>
+                {tagVariant.text}
+              </Tag>
+            </Box>
+
+            <TagAppealRuling
```

Comment:
The appeal ruling tag should not be shown unless the appeal state is `COMPLETED`. Otherwise, the tag may be shown for appeals that are still in progress.

Example:
<img width="1503" alt="image" src="https://github.com/island-is/island.is/assets/795382/4724aa30-0097-4dd4-9b16-9123f08df366">